packages/suite/src/views/firmware/FirmwareModal.tsx

```diff
@@ -81,6 +84,10 @@ export const FirmwareModal = ({
 
     const trezorCancel = () => TrezorConnect.cancel(intl.formatMessage(messages.TR_CANCELLED));
 
+    if (isFirmwareUpdateOnLowBattery) {
+        return <FirmwareLowBatteryModal onClose={toggleFirmwareUpdateOnLowBattery} />;
```

Comment:
dont like the repeated code much..... Maybe move it to global scope?

Reply:
yeah,so the global scope is dead, coz the firmware installation modal doesnt allow dispalying modal over itself...